src/components/Menu/Menu.js

```diff
@@ -21,100 +23,22 @@ import ListItemText from '@material-ui/core/ListItemText';
 import MenuIcon from '@material-ui/icons/Menu';
 import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
 import ChevronRightIcon from '@material-ui/icons/ChevronRight';
-import PermContactCalendar from '@material-ui/icons/PermContactCalendar';
-import EventNote from '@material-ui/icons/EventNote';
-import AccountTree from '@material-ui/icons/AccountTree';
-import Group from '@material-ui/icons/Group';
-import AccountCircle from '@material-ui/icons/AccountCircle';
+import ExitToApp from '@material-ui/icons/ExitToApp';
 
-const drawerWidth = 240;
+import { useStyles } from './Menu.styles';
 
-const menuItems = [
-  {
-    title: 'My Calendar',
-    icon: <PermContactCalendar />,
-    path: '/'
-  }, {
-    title: 'Team Calendar',
-    icon: <EventNote />,
-    path: '/teamCalendar'
-  }, {
-    title: 'Learning Tree',
-    icon: <AccountTree />,
-    path: '/learningTree'
-  }, {
-    title: 'My Team',
-    icon: <Group />,
-    path: '/myTeam'
-  }, {
-    title: 'Profile',
-    icon: <AccountCircle />,
-    path: '/profile'
-  }
-];
-
-const useStyles = makeStyles(theme => ({
-  root: {
-    display: 'flex',
-  },
-  appBar: {
-    transition: theme.transitions.create(['margin', 'width'], {
-      easing: theme.transitions.easing.sharp,
-      duration: theme.transitions.duration.leavingScreen,
-    }),
-  },
-  appBarShift: {
-    width: `calc(100% - ${drawerWidth}px)`,
-    marginLeft: drawerWidth,
-    transition: theme.transitions.create(['margin', 'width'], {
-      easing: theme.transitions.easing.easeOut,
-      duration: theme.transitions.duration.enteringScreen,
-    }),
-  },
-  menuButton: {
-    marginRight: theme.spacing(2),
-  },
-  hide: {
-    display: 'none',
-  },
-  drawer: {
-    width: drawerWidth,
-    flexShrink: 0,
-  },
-  drawerPaper: {
-    width: drawerWidth,
-  },
-  drawerHeader: {
-    display: 'flex',
-    alignItems: 'center',
-    padding: theme.spacing(0, 1),
-    // necessary for content to be below app bar
-    ...theme.mixins.toolbar,
-    justifyContent: 'flex-end',
-  },
-  content: {
-    flexGrow: 1,
-    padding: theme.spacing(3),
-    transition: theme.transitions.create('margin', {
-      easing: theme.transitions.easing.sharp,
-      duration: theme.transitions.duration.leavingScreen,
-    }),
-    marginLeft: -drawerWidth,
-  },
-  contentShift: {
-    transition: theme.transitions.create('margin', {
-      easing: theme.transitions.easing.easeOut,
-      duration: theme.transitions.duration.enteringScreen,
-    }),
-    marginLeft: 0,
-  },
-}));
-
-export default function Menu(props) {
-  const classes = useStyles();
+const Menu = ({ children }) => {
   const theme = useTheme();
-  const [open, setOpen] = useState(false);
+  const classes = useStyles();
   const history = useHistory();
+  const user = useContext(UserContext);
+
+  const [open, setOpen] = useState(false);
+
+  const logout = () => {
+    localStorage.removeItem('token');
+    history.push('/login');
+  }
```

Comment:
I think this should clear the user context too

Reply:
Okay :)